ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -9,14 +8,15 @@ python --version
 python -m virtualenv ./venv_qe
 source ./venv_qe/bin/activate
+cd /tmp
 
 if [[ $TAG == "latest" ]]; then
     LATEST_TAG=$(curl -s "https://api.github.com/repos/cloud-bulldozer/orion/releases/latest" | jq -r '.tag_name');
 else
     LATEST_TAG=$TAG
 fi
-git clone --branch $LATEST_TAG $ORION_REPO --depth 1
-pushd orion
+git clone -q --branch $LATEST_TAG $ORION_REPO --depth 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)